zscan.conf.stop

```diff
@@ -0,0 +1,8 @@
+# Configuration file for setting canbus in Linux host
+
+INTERFACE="$1"
+
+ip link set $INTERFACE down
+
+echo "Removing $INTERFACE"
+rmmod vcan
```

Comment:
We should not `rmmod vcan` here as the module may have other users apart from this script.